rt_bene_model_training/pytorch/util/GenerateRTBENEH5Dataset.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import print_function, division, absolute_import
+
+import argparse
+import os
+
+import h5py
+import numpy as np
+from PIL import Image, ImageFilter, ImageOps
+from torchvision import transforms
+from tqdm import tqdm
+
+script_path = os.path.dirname(os.path.realpath(__file__))
+
+# Augmentations following `prepare_dataset.m`: randomly crop and resize the image 10 times,
+# along side two blurring stages, grayscaling and histogram normalisation
+_required_size = (224, 224)
+_transforms_list = [transforms.RandomResizedCrop(size=_required_size, scale=(0.85, 1.0)),  # equivalent to random 5px from each edge
```

Comment:
Is that really what's happening?!? Can't remember doing that, looks a little crazy .. how is that different from doing a single `RandomResizedCrop` with different scale factor?

Reply:
I'm sorry I don't understand your question...
To be fair, that's somewhat deviating from the original TF code; the scale of (0.85 to 1.0) is a randomized scaling factor as a form of data augmentation...